### PR TITLE
Change automation picker to data table

### PR DESCRIPTION
**Pull request by bramkragten**
image

### CODE REVIEW

**Comment by balloob** [comment]
What do we think of execute in this overview? Should we drop it since it's visible in the more info dialog ?

File: src/components/data-table/ha-data-table.ts [hunk 1]
```diff
@@ -227,10 +227,13 @@ export class HaDataTable extends LitElement {
               const sorted = key === this._sortColumn;
               const classes = {
                 "mdc-data-table__header-cell--numeric": Boolean(
-                  column.type && column.type === "numeric"
+                  column?.type === "numeric"
```
Comment (balloob) [inline src/components/data-table/ha-data-table.ts]: null-chaining is not cheap, it means that it rewrites it to `column && column.type === "numeric"`. So is this necessary? On line 245 we just assume `column` exists.

**Comment by zsarnett** [comment]
I like the ability to quicky execute an automation. 1 less for probably the most import action

File: src/panels/config/automation/ha-automation-picker.ts [hunk 2]
```diff
@@ -42,139 +44,153 @@ class HaAutomationPicker extends LitElement {
...
+                    new Date(automation.attributes.last_triggered),
+                    this.hass.language
+                  )
+                : this.hass.localize("ui.components.relative_time.never")}
+            </div>
+          `,
+        },
+      };
+      if (!narrow) {
+        columns.excecute = {
```
Comment (balloob) [inline src/panels/config/automation/ha-automation-picker.ts]:
```suggestion
        columns.execute = {
```

File: src/panels/config/automation/ha-automation-picker.ts [hunk 3]
```diff
@@ -42,139 +44,153 @@ class HaAutomationPicker extends LitElement {
...
+              "ui.panel.config.automation.picker.show_info_automation"
+            )}"
+          ></paper-icon-button>
+        `,
+      };
+      columns.edit = {
+        title: "",
+        type: "icon-button",
+        template: (_info, automation: any) => html`
+          <paper-icon-button
```
Comment (balloob) [inline src/panels/config/automation/ha-automation-picker.ts]: There is no click listener or `<a>` ?

File: src/panels/config/automation/ha-automation-picker.ts [hunk 4]
```diff
@@ -42,139 +44,153 @@ class HaAutomationPicker extends LitElement {
...
         back-path="/config"
         .route=${this.route}
         .tabs=${configSections.automation}
+        .columns=${this._columns(this.narrow, this.hass.language)}
+        .data=${this._automations(this.automations)}
+        id="entity_id"
+        .noDataText=${this.hass.localize(
+          "ui.panel.config.automation.picker.no_automations"
+        )}
+        @row-click=${this._editAutomation}
```
Comment (balloob) [inline src/panels/config/automation/ha-automation-picker.ts]: I don't think that this is a good experience if there are multiple actions on the row to be able to click on the row too. Means you click next to the execute, you go to edit mode.

Comment (zsarnett) [reply src/panels/config/automation/ha-automation-picker.ts]: I think a double click on the row would be a good experience for getting into the more info dialog along with the (i) icon

**Pull request merged**